dmoj/executors/COFFEE.py

```diff
@@ -2,6 +2,8 @@
 
 from dmoj.executors.script_executor import ScriptExecutor
 
```

Comment:
There shouldn't be any separation here. In general, no separation between DMOJ imports.